apps/customer/views.py

```diff
@@ -81,4 +81,157 @@ def create(self, request, *args, **kwargs):
         return create_instance(self, request, *args, **kwargs)
 
     def update(self, request, *args, **kwargs):
-        return update_instance(self, request, *args, **kwargs)
\ No newline at end of file
+        return update_instance(self, request, *args, **kwargs)
+    
+#==========================================================================
+
+
+def add_key_value_to_all_ordereddicts(od_list, key, value):
+    for od in od_list:
```

Comment:
what is od and od_list here, the variable name should be more understandable

Reply:
Bhagyaraju Updated this method in utils, so I need to import this after his code merged into main.